src/Raven.Studio/typescript/viewmodels/database/settings/databaseSettings.ts

```diff
@@ -72,28 +74,48 @@ class databaseSettings extends viewModelBase {
         });
 
         this.selectedCategory.subscribe(() => this.computeEntriesToShow());
-        this.filterKeys.throttle(500).subscribe(() => this.computeEntriesToShow());
+        
+        this.filterKeys.throttle(500).subscribe(() => {
+            this.computeEntriesToShow();
+            
+            if (this.filterKeys() && this.filteredCategories() && this.filteredCategories().length) {
```

Comment:
and remove first - it think filtered categoryes should be always filled in with valid this - even when no filter then it should match allCategoryNames()

Reply:
done